src/test/java/seedu/address/storage/JsonAdaptedClientTest.java

```diff
@@ -147,6 +147,13 @@ public void toModelType_nullTimezone_throwsIllegalValueException() {
         assertThrows(IllegalValueException.class, expectedMessage, client::toModelType);
     }
 
+    @Test
+    public void toModelType_emptyContractExpiryDate_doesNotThrowException() {
+        JsonAdaptedClient client = new JsonAdaptedClient(VALID_NAME, VALID_PHONE, VALID_EMAIL, VALID_ADDRESS,
+                VALID_COUNTRY, VALID_TIMEZONE, "", VALID_LAST_MODIFIED_INSTANT, VALID_TAGS);
```

Comment:
Should this be a constant `EMPTY_CONTRACT_EXPIRY_DATE`?

Reply:
imo I don't think there's a point in extracting it out as an empty string is kinda like a null value, and we don't extract out null values.